bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/dto/UnitsMap.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2021-2022 Contributors to the SmartHome/J project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.smarthomej.binding.viessmann.internal.dto;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.library.unit.SIUnits;
+import org.openhab.core.library.unit.Units;
+
+/**
+ * This class provides the units for values depending on the viessmann unit
+ * 
+ * @author Ronny Grun - Initial contribution
+ */
+@NonNullByDefault
+public class UnitsMap {
+    private static final Map<String, String> UNIT_MAP = new HashMap<>();
```

Comment:
```suggestion
    private static final Map<String, String> UNIT_MAP = Map.of( //
      "celsius", SIUnits.Celsius.getSymbol(), //
      ...
    );
```

Then you can omit the `static` part below. You can also make the `Map` public and add it to the binding constants class. No need to add a new class for that.